answers.txt

```diff
@@ -0,0 +1,21 @@
+ORDER
+1. SELECT * FROM subjects ORDER BY subject;
+2. SELECT * FROM subjects ORDER BY location;
+
+WHERE
+1.SELECT * FROM books WHERE title = 'Little Women';
+2.SELECT * FROM books WHERE title LIKE '%Python%';
+3.SELECT * FROM subjects WHERE location = 'Main St' ORDER BY subject;
+
+JOINS
+1.SELECT title FROM books JOIN subjects ON books.subject_id = subjects.id WHERE subject = 'Computers';
+2.SELECT title, last_name, first_name, subject FROM books JOIN authors ON books.author_id = authors.id JOIN subjects ON book;
```

Comment:
Good, alternative way to write it for reference - select books.title, authors.first_name, authors.last_name, subjects.subject from books join authors on books.author_id = authors.id join subjects on books.subject_id = subjects.id;